Assets/_BForBoss/_Weapons/Scripts/ScriptableObjects/Weapons/WeaponScriptableObject.cs

```diff
@@ -1,6 +1,7 @@
 using FMODUnity;
 using Sirenix.OdinInspector;
 using UnityEngine;
+using UnityEngine.VFX;
```

Comment:
Don't think this is needed here?

Reply:
You're right, forgot to remove it from when I was testing where to put the VFX code.